DDF-3527 Removed zero width character from pom

rzwiefel opened the pull request:
#### What does this PR do?
Removed zero width character from pom
#### Who is reviewing it? 
anyone

#### Jira Ticket
https://codice.atlassian.net/browse/DDF-3527

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

bdeining commented:
Is there an associated ticket with this?  Seems like this should be tied to a ticket

rzwiefel commented:
@bdeining We added a new Jira ticket type called `Maintainability` that way we have a category to put things that aren't necessarily tech debt. Created ticket for this issue and updated commit.
